feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -127,6 +130,10 @@ fun TestableSnackBar(
     ) {
         val context = LocalContext.current
         scope.launch {
```

Comment:
We shouldn't be using scope.launch within a `@Composable` (the IDE actually complains about this). It's fine for now, but let's follow up to make it more compose idiomatic.